src/Middleware/RetryMiddleware.php

```diff
@@ -44,25 +45,26 @@ class RetryMiddleware implements MiddlewareInterface
 {
     /** @var callable */
     private $nextHandler;
-    private RetrySettings $retrySettings;
-    private ?float $deadlineMs;
+    private Retrier $retrier;
 
-    /*
-     * The number of retries that have already been attempted.
-     * The original API call will have $retryAttempts set to 0.
+    /**
+     * @param callable $nextHandler
+     * @param RetrySettings $retrySettings
+     * @param int $deadlineMillis TODO: Deprecate if possible.
+     * @param int $retryAttempts TODO: Deprecate if possible.
      */
-    private int $retryAttempts;
-
     public function __construct(
         callable $nextHandler,
         RetrySettings $retrySettings,
-        $deadlineMs = null,
+        $deadlineMillis = null,
```

Comment:
Why change the name? I'd rather not change this unless it's necessary, as with named parameters it's possible that it will break something

Reply:
I wanted a uniformity between `RetryMiddleware` and `RetrySettings`. If it will break something, I will remove this change. Thank you for the comment.